application/core/plugins/Authwebserver/Authwebserver.php

```diff
@@ -83,9 +83,9 @@ public function beforeLogin()
                 $this->setAuthPlugin(); // This plugin handles authentication, halt further execution of auth plugins
                 return;
             }
-            if ($this->get('is_default', null, null, $this->settings['is_default']['default'])) {
-                throw new CHttpException(401, 'Wrong credentials for LimeSurvey administration.');
-            }
+        }
```

Comment:
This must be checked only if $this->get('serverkey') is not empty.

Reply:
That was the issue! It was checked only if the server received the proper information.
If the server didn't the regular login was used, not respecting the "default checkbox".

Now, if no login possible with this plugin and the plugin is the defualt, an exception is thrown.

Reply:
Yes but tjis must not happen if `$this->get('serverkey')` is empty : empty not configured :)

Reply:
> Yes but tjis must not happen if $this->get('serverkey') is empty : empty not configured :)

I understand.  But why not?